src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -96,6 +103,50 @@ public RepositoryApiController(ObjectMapper objectMapper, HttpServletRequest req
         return new ResponseEntity<>(studySummary, HttpStatus.CREATED);
     }
 
+    public ResponseEntity<List<JobModel>> enumerateJobs(int offset, int limit){ // should these be optional??
+        List<JobModel> jobModelList = new ArrayList<JobModel>();
+
+        List<FlightState> flightStateList = stairway.getFlights(offset, limit);
+        for (FlightState flightState : flightStateList) {
+            JobModel jobModel = jobService.mapFlightStateToJobModel(flightState);
+            jobModelList.add(jobModel);
+        }
+        return new ResponseEntity<>(jobModelList, HttpStatus.OK);
+    }
+
+    public ResponseEntity<JobModel> retrieveJob(String jobId){
+        FlightState flightState = stairway.getFlightState(jobId);
```

Comment:
My suggestion would be to move the logic for these job endpoints into the JobService class and make this code very simple. That keeps Stairway wrapped by JobService. Not a big deal, but a little tidier.